packages/dialtone-css/lib/build/less/components/toast.less

```
@@ -119,3 +119,145 @@
+  font-size: var(--toast-alternate-font-size);
+  line-height: var(--toast-alternate-line-height);
+  word-break: normal;
+  background-color: var(--toast-alternate-color-background);
+  border: var(--toast-alternate-border-width) var(--toast-alternate-border-style) var(--toast-alternate-border-color);
+  border-radius: var(--toast-alternate-border-radius);
+  box-shadow: var(--toast-alternate-box-shadow);
+  pointer-events: all;
+
+  //  If there's a link in a toast, break the url so it wraps
+  > a {
+    word-break: break-all;
+  }
+
+  // If you want to hide and reveal the toast
+  &[aria-hidden='true'] {
+    visibility: hidden;
+    opacity: var(--dt-opacity-0);
+  }
+
+  &[aria-hidden='false'] {
+    visibility: visible;
+    opacity: var(--dt-opacity-1300);
+  }
+
+  .d-toast-layout-alternate__icon {
+    display: flex;
+    flex: 0 auto;
+    margin-right: var(--dt-space-300);
+    color: var(--toast-alternate-color-icon);
+  }
+
+  .d-notice__title {
+    color: var(--dt-color-foreground-muted);
+    font-weight: var(--dt-font-weight-normal);
+    font-size: var(--dt-font-size-100);
+    word-break: break-all;
+  }
+
+  .d-notice__actions .base-button__button {
+    padding: var(--dt-size-100);
+    color: var(--dt-action-color-foreground-muted-default);
+  }
+}
+
+//  ============================================================================
+//  $   CONTENT CONTAINER
+//  ----------------------------------------------------------------------------
+.d-toast-alternate__content {
+  display: flex;
+  flex: 1 auto;
+  flex-direction: column;
+  margin-right: var(--dt-space-500);
+  color: var(--dt-color-foreground-primary);
+  font-weight: var(--dt-font-weight-normal);
+  font-size: var(--dt-font-size-200);
+  word-break: break-all;
+}
+
+.d-toast-alternate__dialog {
+  position: relative;
+  align-items: center;
+
+
+  .d-notice__actions button:first-child {
+      margin-left: var(--dt-space-600); // 32
+  }
+}
+
+.d-toast-alternate__header {
+  display: flex;
+  gap: var(--dt-space-200);
+  align-items: center;
+  font-color: var(--toast-alternate-font-color);
+}
+
+//  $$  ERROR
+//  ----------------------------------------------------------------------------
+.d-toast-alternate--error {
+    --toast-alternate-border-color: var(--dt-color-border-critical-subtle);
+}
+
+//  $$  INFO
+//  ----------------------------------------------------------------------------
+
+.d-toast-alternate--info {
+    --toast-alternate-border-color: var(--dt-color-border-focus);
+}
+
+//  $$  SUCCESS
+//  ----------------------------------------------------------------------------
+.d-toast-alternate--success {
+    --toast-alternate-border-color: var(--dt-color-border-success-subtle);
+}
+
+//  $$  GRADIENT
+//  ----------------------------------------------------------------------------
+.d-toast-alternate--gradient {
+  background: linear-gradient(var(--dt-color-neutral-white), var(--dt-color-neutral-white)) padding-box,
```

Use surface color to make it work on dark/light theme

```suggestion
  background: linear-gradient(var(--dt-color-surface-primary), var(--dt-color-surface-primary) padding-box,
```